sympy/polys/domains/gaussiandomains.py

```diff
@@ -276,7 +274,7 @@ def get_ring(self):
 
     def get_field(self):
         """Returns a field associated with ``self``. """
-        return QQ_I
+        return ZZ_I
```

Comment:
This should be the *field*, i.e. `QQ_I`.

Reply:
The field of ZZ_I is QQ_I and the field of QQ_I is ZZ_I (as you had it before)?

Reply:
`ZZ_I` is the *ring* of `QQ_I`. It should be returned by `QQ_I.get_ring()`.